### PR TITLE
PythonDjango Sample

### PR DESCRIPTION
**Pull request by dodaromike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,91 @@
...
+page_type: sample
+description: This sample demonstrates how to use the Microsoft Graph to access data in Office 365 from a Python Django app.
+products:
+- ms-graph
+- microsoft-graph-calendar-api
+- office-exchange-online
+languages:
+- python
+---
+# Build Python Django apps with Microsoft Graph
```
Comment (jasonjoh) [inline README.md:11]: Update the title: Microsoft Graph sample Python web app

File: README.md [hunk 2]
```diff
@@ -1,21 +1,91 @@
...
 
-This module will introduce you to working with Microsoft Graph to access data in Office 365 by building a Python Django web applications.
+This sample uses Microsoft Graph to access data in Office 365 by building a Python Django web application using the Azure AD v2 authentication endpoint.
 
-## Lab - Build Python Django apps with Microsoft Graph
+- [Python Microsoft Graph tutorial](https://docs.microsoft.com/graph/tutorials/python)
```
Comment (jasonjoh) [inline README.md:15]: Remove this line

File: README.md [hunk 3]
```diff
@@ -1,21 +1,91 @@
...
-If you just want the completed sample generated by following this lab, you can find it here.
+## Register a web application with the Azure Active Directory admin center
 
-- [Completed project](demo)
+1. Open a browser and navigate to the [Azure Active Directory admin center](https://aad.portal.azure.com). Login using a **personal account** (aka: Microsoft Account) or **Work or School Account**.
 
-## Disclaimer
+1. Select **Azure Active Directory** in the left-hand navigation, then select **App registrations** under **Manage**.
+
+    ![A screenshot of the App registrations ](/tutorial/images/aad-portal-app-registrations.png)
```
Comment (jasonjoh) [inline README.md:32]: Remove all of the screenshots from this section.

**Review by jasonjoh — CHANGES_REQUESTED** [review]
Good start, but it looks like this is a work in progress? The project isn't complete.

**Comment by dodaromike** [comment]
Copied the project files to branch for review. Ready for review.

**Review by jasonjoh — APPROVED** [review]

**Pull request merged**